config/routes.rb

```diff
@@ -8,8 +8,7 @@
   get '/signin' => 'sessions#new', :as => :signin
   get '/signout' => 'sessions#destroy', :as => :signout
 
+  get '/orgs' => 'orgs#index', :as => :orgs
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.